app/views/registrations/new.html.erb

```diff
@@ -1,8 +1,24 @@
+
 <h1>Registration</h1>
 <%= form_for @voter, url: url_for(controller: 'registrations', action: 'create') do |f| %>
+ <script>
+            function Color(value, dark){ 
+    document.body.style.backgroundColor = '#' + value; 
+    if (dark) {
+        document.body.style.color = '#FFFFFF'; }
+        }
+        </script>
+        
   <div>
-      <h2 class="name">High School Name</h2>
-      <%= f.select :school_id, options_for_select(@schools) %>
+   <h2 class="name">High School Name</h2>
+      <select name="highschool" onChange="Color(this.value)"> 
+        <option value = FFFFFF> Please Select a High School</option>
+        <option value="FF0000">Shaw High School </option>
+      	<option value="761817">Jane Addams High School</option>
+      	<option value="FFFFFF">John Marshall High School</option>
+      	<option value="7B0D8C">Westshore High School</option>
+      	<option value="1532ED">Beaumont High School</option>
```

Comment:
You'll need to retain the school id values here.  Otherwise, it won't be able to save the school id when submitted.  Maybe you could create a mapping of the school_id => colors in js?
```
$(function() {
  var colorMappings = {};
  colorMappings["Shaw High School"] = "FF000";
  colorMappings["Jane Addams High School"] = "761817";
  // and so on
  $('#voter_school_id').change(function() {  
    var school = $('#voter_school_id option:selected').text();
    document.body.style.color = colorMappings[school];
  });
});

Then you can just leave the databound select element as it was before.